addons/mail/static/tests/helpers/mock_server/models/res_partner.js

```diff
@@ -272,12 +272,8 @@ patch(MockServer.prototype, {
                         is_company: partner.is_company,
                         name: partner.name,
                         type: "partner",
-                        user: mainUser
-                            ? {
-                                  id: mainUser.id,
-                                  isInternalUser: !mainUser.share,
-                              }
-                            : false,
+                        userId: mainUser ? mainUser.id : undefined,
+                        isInternalUser: mainUser ? !mainUser.share : undefined,
```

Comment:
Inconsistent with python (either should be false, or not have the keys at all).

Reply:
It's more LOCs to make sure the keys are not present, more diff... `undefined` means roughly the same